terraform/060-simplesamlphp/vars.tf

```diff
@@ -5,13 +5,13 @@ variable "secret_salt" {
 }
 
 variable "memory" {
-  type    = string
-  default = "96"
+  type    = number
```

Comment:
Might be helpful to add description here too with default in case it changes (for both this and cpu).